bindings/python/src/tokenizer.rs

```diff
@@ -53,7 +53,7 @@ use crate::processors::PyPostProcessor;
 ///         Yesterday"``.
 ///
 #[pyclass(dict, module = "tokenizers", name=AddedToken)]
-#[text_signature = "(content, single_word=False, lstrip=False, rstrip=False, normalized=True)"]
+#[text_signature = "(self, content, single_word=False, lstrip=False, rstrip=False, normalized=True)"]
```

Comment:
I think we should probably remove all the `self` everywhere and document all the pyclass using `__new__` instead. (I guess its the reason you add them in the first place?). This would be more accurate, what do you think?

Reply:
Well all code was previously documenting __init__. I feel it's more `pythonic` than  __new__ even though it's really a `__new__` . `self` would become `cls` then.